### DISCUSSION TITLE
Directory preview straight line

### DISCUSSION THREAD

**Discussion by 4zv4l** [sxyazi/yazi#1214] [1. Q&A] [answered]
Hi,
When previewing a directory:
![image](https://github.com/sxyazi/yazi/assets/46655455/8614a45a-ea85-415a-8952-2735517bd7f3)
there is a straight line that shows where the cursor will be when entering the directory (sometimes not the first entry), is it possible to modify this so it is always the first entry and/or to hide this straight line or make it smaller (just underline the entry and not the whole length of the preview pane)

**Comment by lpnh** [comment, marked as answer]
Hi! You can hide it by adding the following to your `theme.toml` file:

```toml
[manager]
preview_hovered = { underline = false }
```

You can find more information about the available options [here](https://yazi-rs.github.io/docs/configuration/theme).

**Reply by 4zv4l** [reply]
Thank you so much !
